docs/api.rst

```diff
@@ -547,19 +580,19 @@ mergeConcurrently
 Given a higher-order stream, return a new stream that merges inner streams as
 they arrive up to the specified concurrency. Once concurrency number of streams
 are being merged, newly arriving streams will be merged after an existing one
-ends.::
+ends. ::
 
   s:                            --a--b--c--d--e-->
   t:                            --x------y|
   u:                            -1--2--3--4--5--6>
   stream:                       -s--t--u--------->
   mergeConcurrently(2, stream): --a--b--cy4d-5e-6>
 
-Note that u is only merged after t ends, due to the concurrency level of 2.
+Note that ``u`` is only merged after ``t`` ends because of the concurrency level of 2.
```

Comment:
TIL ["due to" vs "because of"](http://linguistech.ca/Tips+and+Tricks+-+Because+of+vs.+Due+to) :+1: